battles/models.py

```diff
@@ -20,6 +20,7 @@ def __str__(self):
             self.opponent.get_short_name()
         )
 
+    @property
```

Comment:
Since this is a getter property, it'd be better named only `status_label` instead of `get_status_label`.